docs/install/proxmox/proxmox-setup.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Installing Colony with Promox
+description: Follow the steps to install Colony with Proxmox
+sidebar_position: 2
+---
+
+## Installing Colony with Proxmox
+
+This guide walks through the setup of Laptop-0, a machine for running Colony, and 3 additional VM's for Colony to auto discovery and install a K3S cluster.  Some items to note:
+
+- Colony requires Docker.
+- If you have a firewall on your host machine (laptop 0) it needs to allow the following ports (ports TBA).
+
+Warning: Colony will be able to PXE boot all machines in your network.
+
+## Getting Started
+
+The steps here are optional, but we recommend completing these to create a Proxmox template VM that you can clone to for better performance and asset creation.
+
+### Debian image
+
+These steps use a `debian cloudinit image for laptop VM`.
+
+- You will need to install `libguestfs-tools` to interact with cloud-init image.
+
+    ```shell
+    apt update
+    apt install libguestfs-tools
+    ```
+
+- Commands should be run in Proxmox shell, which can be accessed through the Proxmox UI through `datacenter > <node-name> > shell`, or through ssh.
+
+- Might to prepend sudo if you running user different than `root`
+
+Download debian cloud init image [debian download image](https://cloud.debian.org/images/cloud/). Create the directory anywhere. Select the correct architecture. This example runs on x86
+
+```shell
+mkdir debian-golden-img
+
+wget https://cloud.debian.org/images/cloud/bookworm/20241004-1890/debian-12-genericcloud-amd64-20241004-1890.qcow2
+
+```
+
+### Guest agent and VIM
+
+Install `qemu-guest-agent` and `vim`. This enables information to report back from the guest machine to the host. Allows power off commands to pass to the guest.
```

Comment:
```suggestion
We will use `virt-customize` to install `qemu-guest-agent` and `vim`. The qemu-guest-agent is used to provide a communication channel between a virtual machine and its Proxmox. enabling features like shutdown, time synchronization, and better management of guest systems.
```

Reply:
`qemu-guest-agent` should also be a code item.